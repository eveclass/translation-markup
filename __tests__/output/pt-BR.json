{
  "SYSTEM": {
    "COPY": "Copiar",
    "ABOUT": "Sobre",
    "DASHBOARD": "Dashboard"
  }
}