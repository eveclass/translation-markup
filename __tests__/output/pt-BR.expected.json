{
  "SYSTEM": {
    "ABOUT": "Sobre",
    "COPY": "Copiar",
    "DASHBOARD": "Dashboard"
  }
}
